### PR TITLE
Fe UI all submisisons

### PR DESCRIPTION
**Pull request by vicscherman**
### What this PR does (required):
Setting up front end styling for viewing art submitted to a contest you own

### Screenshots / Videos (required):

https://user-images.githubusercontent.com/69362301/121976384-b1904800-cd51-11eb-9dbe-fafb8365fdae.mp4

### Any information needed to test this feature (required):
Added a temporary protected route/button to access and test this. Click the "View submissions" button to test

### Any issues with the current functionality (optional):
right now have a temporary variable called "contestOwnerId" that is used to test the conditional rendering. This will need to come from the database/logged in user in the finalized integration

### CODE REVIEW

File: client/src/pages/Dashboard/Dashboard.tsx [hunk 1]
```diff
@@ -13,6 +13,7 @@ import Profile from '../Profile/Profile';
 import Submission from '../Submission/Submission';
 import Contest from '../Contest/Contest';
 import EditProfile from '../Profile/EditProfile/EditProfile';
+import ViewContestSubmissions from '../ViewContestSubmissions/ViewContestSubmissions';
```
Comment (dproc96) [inline client/src/pages/Dashboard/Dashboard.tsx:16]: Can you add a protected route for this page and a way to navigate to it? Maybe via the profile dropdown menu

File: client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
+import react, { useState, useEffect } from 'react';
+import { Grid, Typography, Box, Chip, Avatar } from '@material-ui/core';
+import GridList from '@material-ui/core/GridList';
+import GridListTile from '@material-ui/core/GridListTile';
+import GridListTileBar from '@material-ui/core/GridListTileBar';
+import IconButton from '@material-ui/core/IconButton';
+import AddCircleOutlineIcon from '@material-ui/icons/AddCircleOutline';
+import useStyles from './useStyles';
+
+import { contest, contests } from '../../interface/tempContestData';
```
Comment (dproc96) [inline client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx:10]:
```suggestion
import { Contest, Contests } from '../../interface/tempContestData';
```
Would be better practice to have these interfaces capitalized

File: client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  { img: 'https://source.unsplash.com/1600x900/?grim,reaper', author: 'Sarah', title: 'you are my friend' },
+  { img: 'https://source.unsplash.com/1600x900/?brazil,country', author: 'Paul', title: 'what is up' },
+  { img: 'https://source.unsplash.com/1600x900/?forest,fight', author: 'Milad', title: 'hello' },
+  { img: 'https://source.unsplash.com/1600x900/?tank,fighter', author: 'Carlos', title: 'hello' },
+];
+
+export default function ViewContestSubmissions(): JSX.Element {
+  const classes = useStyles();
+  const [contestCard, setContestCard] = useState<contest>(Object);
+  const [winnerIndex, setWinnerIndex] = useState(Number);
```
Comment (dproc96) [inline client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx:29]: I think right now you are setting these default values to classes of Object and Number, not to an instance of them. Would be better to do something like this:
```suggestion
  const [contestCard, setContestCard] = useState<contest>(null);
  const [winnerIndex, setWinnerIndex] = useState(-1);
```
And in the case of the winner index, have some logic to check if it equals 0 or greater

File: client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+  };
+
+  useEffect(() => {
+    handleContest();
+  }, [contestCard]);
+
+  //   this is the grossest thing i've ever coded but it's just there to test functionality
+  useEffect(() => {
+    setWinnerIndex(999);
+  }, []);
```
Comment (dproc96) [inline client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx:43]: Not really necessary to include this, better to just set an initial value above

File: client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+
+  const handleWinnerIndex = (index: number) => {
+    setWinnerIndex(index);
+    console.log(winnerIndex);
```
Comment (dproc96) [inline client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx:47]: You can remove this console log

File: client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+                  <Chip label="Winner"></Chip>
+                </IconButton>
+              ) : (
+                <IconButton
+                  key={index}
+                  aria-label={`info about ${tile.title}`}
+                  onClick={() => handleWinnerIndex(index)}
+                  className={classes.icon}
+                >
+                  <AddCircleOutlineIcon />
```
Comment (dproc96) [inline client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx:75]: Maybe something like a trophy icon would make more sense? I don't know if I would intuitively assume this was the choose winner button as is

**Review by dproc96 — COMMENTED** [review]
Good start and the functionality is there, just had a few notes on implementation

File: client/src/pages/Dashboard/Dashboard.tsx (hunk 1 above)
Comment (vicscherman) [reply client/src/pages/Dashboard/Dashboard.tsx:16]: Added a button for now

File: client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx (hunk 3 above)
Comment (vicscherman) [reply client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx:29]: Was able to default winnerIndex to -1 but setContest card needed to be Object

File: client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx (hunk 6 above)
Comment (vicscherman) [reply client/src/pages/ViewContestSubmissions/ViewContestSubmissions.tsx:75]: Done

**Pull request merged**